protractor.conf.js

```diff
@@ -1,21 +1,28 @@
 // See https://github.com/angular/protractor/blob/master/docs/referenceConf.js
 // for full protractor config reference.
+
+function getJobName(browser) {
+  var branchAndCommit =
+      process.env.TRAVIS_BRANCH + '-' + process.env.TRAVIS_COMMIT;
+
+  return process.env.TRAVIS_PULL_REQUEST == 'false' ?
+             'CO-' + browser + '-' + branchAndCommit :
+             'PR-' + browser + '-' + process.env.TRAVIS_PULL_REQUEST + '-' +
+                 branchAndCommit;
+}
+
 exports.config = {
   sauceUser: process.env.SAUCE_USERNAME,
 
   sauceKey: process.env.SAUCE_ACCESS_KEY,
 
-  // Options specific to which browser tests are run on.
-  capabilities: {
-    'browserName': 'chrome',
-    'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
-    'build': process.env.TRAVIS_BUILD_NUMBER,
-    'name': process.env.TRAVIS_PULL_REQUEST == 'false' ?
-                'CO-' + process.env.TRAVIS_BRANCH + '-' +
-                    process.env.TRAVIS_COMMIT :
-                'PR-' + process.env.TRAVIS_PULL_REQUEST + '-' +
-                    process.env.TRAVIS_BRANCH + '-' + process.env.TRAVIS_COMMIT
-  },
+
+  multiCapabilities: [
```

Comment:
Sorry, ignore this PR for now. I create it just to test an end-to-end run on Sauce/Travis w/ "multiCapabilities". It's the only way to (AFAIK) to do it.